src/deterministic_vi/elbo.jl

```diff
@@ -235,11 +235,12 @@ function accum_star_pos!{NumType <: Number}(
                     x::Vector{Float64},
                     wcs_jacobian::Array{Float64, 2},
                     calculate_derivs::Bool)
+
     eval_bvn_pdf!(elbo_vars.bvn_derivs, bmc, x)
 
-    # TODO: Also make a version that doesn't calculate any derivatives
-    # if the object isn't in active_sources.
-    get_bvn_derivs!(elbo_vars.bvn_derivs, bmc, true, false)
+    if elbo_vars.calculate_derivs && calculate_derivs
```

Comment:
No.  If you look at how ```calculate_derivs``` is defined, it is determined by whether this is an active source.  The variable ```elbo_vars.calcualte_derivs``` determines whether we are calculating derivatives at all in this ELBO evaluation.  Please compare with the identical usage five lines later.